DRILL-5765: Json query profile is not shown on Web UI

sindhurirayavaram opened the pull request:
The "graph.js", used in the profiles page of Drill Web UI, uses $(window).load(function () {}). The ".load" function is deprecated from the Jquery after 1.8. In this change, I have changed the function to onload.

arina-ielchiieva commented:
@sindhurirayavaram can you please explain why your previous changes caused the regression and how they are connected with `load` function. Thank you in advance.

sindhurirayavaram commented:
@arina-ielchiieva I updated the jquery to it's latest min version. The updated jquery library doesn't support load function anymore. It should be "onload". We were using load in graph.js which was failing. I updated that to "onload".

arina-ielchiieva commented:
Thanks for explanation. +1
